python/ray/tests/test_placement_group.py

```diff
@@ -791,5 +792,60 @@ def ping(self):
     assert all(ray.get([a.ping.remote() for a in actors]))
 
 
+def test_capture_child_tasks(ray_start_cluster):
+    cluster = ray_start_cluster
+    total_num_actors = 4
+    for _ in range(2):
+        cluster.add_node(num_cpus=total_num_actors)
+    ray.init(address=cluster.address)
+
+    pg = ray.util.placement_group(
+        [{
+            "CPU": 2
+        }, {
+            "CPU": 2
+        }], strategy="STRICT_PACK")
+    ray.get(pg.ready(), timeout=1)
+
+    # If get_current_placement_group is used when the current worker/driver
+    # doesn't belong to any of placement group, it should raise an exception.
+    with pytest.raises(RuntimeError):
+        get_current_placement_group()
+
+    @ray.remote(num_cpus=1)
+    class NestedActor:
+        def ready(self):
+            return True
+
+    @ray.remote(num_cpus=1)
+    class Actor:
+        def __init__(self):
+            self.actors = []
+
+        def ready(self):
+            return True
+
+        def schedule_nested_actor(self):
+            actor = NestedActor.options(
+                placement_group=get_current_placement_group()).remote()
```

Comment:
Shall we make capture the default behavior by automatically calling get_current_placement_group() if placement group is not otherwise set?

Reply:
Yeah, that's a pretty elegant solution. But, in that case, it will be tricky to support the `capture_child_tasks` argument right? I am fine with postponing to add that argument until users complain though. What do you think?

Reply:
(Will address it after your next comment).

Reply:
If we do that, I think the whole issue can be just closed.

Reply:
I was thinking you can set capture_child_tasks=False to avoid that behavior, otherwise we capture by default.